src/main/java/com/example/onlinebookstore/errors/BookErrorDto.java

```diff
@@ -0,0 +1,10 @@
+package com.example.onlinebookstore.errors;
+
+import java.time.LocalDateTime;
+import org.springframework.http.HttpStatus;
+
+public record BookErrorDto(
+        HttpStatus status,
+        LocalDateTime timestamp,
+        Object errorMessage) {
```

Comment:
if it is message - it should be `String`

if it is not only message - consider renaming to `errorPayload`